pkg/koordlet/util/idleinfo_kidled.go

```diff
@@ -0,0 +1,192 @@
+/*
+Copyright 2022 The Koordinator Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package util
+
+import (
+	"os"
+	"path/filepath"
+	"reflect"
+	"strconv"
+	"strings"
+
+	"k8s.io/klog/v2"
+
+	"github.com/koordinator-sh/koordinator/pkg/koordlet/util/system"
+)
+
+var (
+	KidledScanPeriodInSecondsFilePath = filepath.Join(system.Conf.SysRootDir, "/kernel/mm/kidled/scan_period_in_seconds")
```

Comment:
1. please move the kidled utils under the pkg/koordlet/util/system
2. generate the path in the runtime since the `SysRootDir` can be changed after the initialization

Reply:
This problem is not resolved. You can define the sub filepath as const and generate the full path in the runtime after the `system.Conf.SysRootDir` is parameterized.